flow_screen_components/ChatSessionManagerSC/ChatSessionManagerSCHelper.js

```diff
@@ -0,0 +1,80 @@
+({
```

Comment:
Is this exactly the same code as ChatSessionManagerLAHelper.js ? If that's the case, I think that I would change things so that the action and screen component live in the same project, and they both import the same javascript file, so that you only have to have one version of this code. ...

Reply:
I think I would consolidate it in the flow_screen_components/ChatSessionManager directory. Then we can package it in a single package with both the action and the screen component.

Reply:
My initial objective was to have one component. What I noticed and lead me to create two separate components:

- With one component you should have two methods in the controller "init" (for the screen component) and "invoke" (for the local action)

- When using the component as an Action both methods "init" and "invoke" are called
So I split  the component in two separate components